repair/repair_writer.hh

```diff
@@ -45,6 +45,19 @@ public:
             , _reason(reason) {
     }
 
+    void start_streaming(sharded<replica::database>& _db, sharded<db::system_distributed_keyspace>& _sys_dist_ks, sharded<db::view::view_update_generator>& _view_update_generator) {
+        create_writer(_db, _sys_dist_ks, _view_update_generator);
```

Comment:
What's the point of making the "create_writer" function private, and then writing a new function "start_streaming" that is public and just calls it? Why not keep the original "create_writer" public?

Reply:
I agree. This new start_streaming is also confusing.  It is not starting streaming but just create a write.

Reply:
That's a remnant of my thought process, shouldn't have made it to the patch. The only relevant bit is done a bit later when I remove the parameters from this function and move them to the constructor. Will clean up.